client/transactions/list/test/index.tsx

```diff
@@ -555,10 +555,12 @@ describe( 'Transactions list', () => {
 				)
 			).not.toBe( -1 ); // amount
 			expect(
-				getUnformattedAmount( displayFirstTransaction[ 3 ] ).indexOf(
```

Comment:
I had to change the approach of this assertion. If the fee amount is a float, in the list it is displayed as '$0.50' whereas in the CSV it is formatted as '0.5' (note the absence of trail zero). We can get rid of the currency using the `getUnformattedAmount` function, that returns '0.50' in this example. To overcome this, I decided to convert '0.50' and '0.5' to Number.
Please let me know if you think there is better way to do this.